entities/entities/spawned_money/cl_init.lua

```diff
@@ -3,27 +3,25 @@ include("shared.lua")
 function ENT:Initialize()
 end
 
+local vector_up = Vector(0,0,1)
 function ENT:Draw()
     self:DrawModel()
+    local offset = (self:OBBMaxs() - self:OBBCenter())*vector_up
```

Comment:
I use notepad++, if you really want to keep the entire DarkRP gamemode that complaint to the specs you defined in glualint, you can edit it yourself if you'd like.